src/main/java/ai/datahunters/md/server/analytics/AnalyticsData.java

```diff
@@ -0,0 +1,14 @@
+package ai.datahunters.md.server.analytics;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
```

Comment:
Please make it a value, it add some more nice features, which could be quite useful
- hashCode and equals
- toString